### PR TITLE
fixes #1448, CTRL+C and CTRL+A works now

### LINKED ISSUE #1448: [Regression] ctrl + c doesn't work

### PR DESCRIPTION
**Pull request by kushagrasurana**
Dont know why QShortcut is not working.

### CODE REVIEW

**Comment by kushagrasurana** [comment]
@dubslow 
nice idea, updated.

File: src/chatlog/chatlog.cpp [hunk 1]
```diff
@@ -830,3 +828,13 @@ void ChatLog::focusOutEvent(QFocusEvent* ev)
             lines[i]->selectionFocusChanged(false);
     }
 }
+
+void ChatLog::keyPressEvent(QKeyEvent * event)
+{
+    if(event->matches(QKeySequence::Copy))
```
Comment (dubslow) [inline src/chatlog/chatlog.cpp]: Put a space between the `if` and the paren, as per the newly-refurbished code style guide on the wiki :)

**Comment by zetok** [comment]
@kushagrasurana if you could in the future contributions: https://github.com/tux3/qTox/wiki/Coding

File: src/chatlog/chatlog.cpp [hunk 2]
```diff
@@ -830,3 +828,13 @@ void ChatLog::focusOutEvent(QFocusEvent* ev)
...
+        if(copyAction->isEnabled())
```
Comment (dubslow) [inline src/chatlog/chatlog.cpp:836]: Actually @zetok I've decided that nested `if`s must always be braced, for if there's any `else` statement, confusion _will_ arise.

@kushagrasurana Please fix that here as well.

**Comment by kushagrasurana** [comment]
@zetok sure, i will remember the guideline.

**Comment by zetok** [comment]
Fixes issue(s) for me.

@dubslow when it will be merged?

**Comment by Jookia** [comment]
There's still broken shortcuts in the chat forms.

**Comment by Jookia** [comment]
#1457 fixes this properly.

**Comment by dubslow** [comment]
Yep, closed in favor of 1457 which I just merged

**Pull request closed without merging**